coroutines-interop/src/nativeCommonMain/kotlin/com/badoo/reaktive/coroutinesinterop/LaunchCoroutine.kt

```diff
@@ -14,7 +15,7 @@ internal actual inline fun <T> launchCoroutine(
     crossinline onError: (Throwable) -> Unit,
     crossinline block: suspend CoroutineScope.() -> T
 ) {
-    val disposable = Disposable()
+    val disposable = disposable()
```

Comment:
Why do you have such changes?

Reply:
Looks like in `10dad76` he did mistake in branch merging, and reverted it in `bcc830f`. So previous implementation is kept and branch is merged.

Looks like it is required to checkout `e7176b5`, merge `master` again and force push branch.

Reply:
> Why do you have such changes?

I think I made a mistake when merging.